dev/com.ibm.ws.anno/resources/com/ibm/ws/anno/resources/internal/AnnoMessages.nlsprops

```diff
@@ -222,33 +222,33 @@ ANNO_CLASSINFO_CLASS_NOTFOUND.useraction=Verify that the class is included in th
 # # {2} The error message from the exception
 ANNO_INFOSTORE_OPEN1_EXCEPTION=CWWKC0026W: [ {0} ]: The open of [{1}] caused an exception. The message is: {2}
 ANNO_INFOSTORE_OPEN1_EXCEPTION.explanation=The open of a class source caused an exception.
-ANNO_INFOSTORE_OPEN1_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_INFOSTORE_OPEN1_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
 
 # # {0} A hashcode id used to identify the object emitting the message
 # # {1} A hashcode id used to identify the object causing the exception
 ANNO_INFOSTORE_CLOSE1_EXCEPTION=CWWKC0027W: [ {0} ]: The close of [{1}] caused an exception.
 ANNO_INFOSTORE_CLOSE1_EXCEPTION.explanation=The close of a class source caused an exception.
-ANNO_INFOSTORE_CLOSE1_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_INFOSTORE_CLOSE1_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
 
 # # {0} A hashcode id used to identify the object emitting the message
 # # {1} the full resource name for the class
 # # {2} The class name
 ANNO_INFOSTORE_OPEN2_EXCEPTION=CWWKC0028W: [ {0} ]: The open of an input stream for resource [{1}] for class [{2}] failed with an exception.
 ANNO_INFOSTORE_OPEN2_EXCEPTION.explanation=When the annotation scan process tried to open an input stream for the class, an exception occurred.
-ANNO_INFOSTORE_OPEN2_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_INFOSTORE_OPEN2_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
 
 # # {0} A hashcode id used to identify the object emitting the message
 # # {1} the full resource name for the class
 # # {2} The class name
 ANNO_INFOSTORE_CLOSE2_EXCEPTION=CWWKC0029W: [ {0} ]: The close of an input stream for resource [{1}], class [{2}], failed with an exception.
 ANNO_INFOSTORE_CLOSE2_EXCEPTION.explanation=When the annotation scan process tried to close an input stream for the class, an exception occurred.
-ANNO_INFOSTORE_CLOSE2_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_INFOSTORE_CLOSE2_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
```

Comment:
Change
'For more information on the topic, see the exception and the associated FFDC message.'
to
'For more information, see the exception and the associated FFDC message.'

Reply:
Updated.